Lumina/Lumina/UI/Extensions/Delegates/LuminaDelegate.swift

```diff
@@ -72,6 +72,10 @@ public protocol LuminaDelegate: class {
     ///
     /// - Parameter controller: the instance of Lumina that cancel was tapped on
     func dismissed(controller: LuminaViewController)
+    
+    /// Triggered whenever the image is tapped (outside of the buttons)
```

Comment:
Thank you for adding the documentation comment here :) if we make this change, can you also update the README?